reference/performance/statement-summary.md

```diff
@@ -123,28 +230,43 @@ The statement summary feature is disabled by default. You can enable it by setti
 set global tidb_enable_stmt_summary = true;
 ```
 
-The `tidb_enable_stmt_summary` system variable has two scopes - global and session, which work a little differently from other system variables, as described below:
+The statistics in the system table will be cleared if the statement summary feature is disabled, and will be re-calculated next time the statement summary feature is enabled. Tests have shown that enabling this feature has little impact on performance.
+
+Another two system variables that control the statement summary:
+
+- `tidb_stmt_summary_refresh_interval`: The interval at which the `events_statements_summary_by_digest` table is refreshed.The time unit is second (s). The default value is `1800`.
+- `tidb_stmt_summary_history_size`: The size of each SQL statement category historically stored in the `events_statements_summary_by_digest_history` table. The default value is `24`.
+
+The following is a configuration example of statement summary:
+
+{{< copyable "sql" >}}
+
+```sql
+set global tidb_stmt_summary_refresh_interval = 1800;
+set global tidb_stmt_summary_history_size = 24;
+```
+
+When the above configuration takes effect, every 30 minutes the `events_statements_summary_by_digest` is cleared. `events_statements_summary_by_digest_history` stores data generated over the recent 12 hours.
+
+The above two system variables have two scopes - global and session, which work a little differently from other system variables, as described below:
 
 - Set the global variable to apply to the cluster immediately.
 - Set the session variable to apply to the current TiDB server immediately. This is useful when you debug on a single TiDB server instance.
 - The session variable has a higher read priority. The global variable will be read only if no session variable is set.
 - Set the session variable to a blank string to re-read the global variable.
 
-The statistics in the system table will be cleared if the statement summary feature is disabled, and will be re-calculated next time the statement summary feature is enabled. Tests have shown that enabling this feature has little impact on performance.
-
-`events_statements_summary_by_digest` is a memory table. To prevent potential memory issues, we need to limit the number of statements to be saved and the longest SQL display length. You can configure these limits using the following parameters under `[stmt-summary]` of `config.toml`:
+The statement summary tables are memory tables. To prevent potential memory issues, we need to limit the number of statements to be saved and the longest SQL display length. You can configure these limits using the following parameters under `[stmt-summary]` of `config.toml`:
```

Comment:
```suggestion
The statement summary tables are memory tables. To prevent potential memory issues, you need to limit the number of statements to be saved and the longest SQL display length. You can configure these limits using the following parameters under `[stmt-summary]` of `config.toml`:
```